### PR TITLE
hax: fix SIGSEGV panic on CentOS 8

### PR DESCRIPTION
**Pull request by andriytk**
The following panic is observed on CentOS 8 on startup:

    hare-hax[270375]: 2021-11-08 16:04:42,950 [DEBUG] {MainThread} Loading library from path: /opt/seagate/cortx/hare/lib64/python3.6/site-packages/hax/motr/../../libhax.cpython-36m-x86_64-linux-gnu.so
    hare-hax[270375]: 2021-11-08 16:04:43,061 [INFO] {qconsumer-0} Handler thread has started
    hare-hax[270375]: 2021-11-08 16:04:43,061 [DEBUG] {qconsumer-0} Adopting Motr thread
    hare-hax[270375]: motr[270375]:  c700  FATAL  [lib/assert.c:50:m0_panic]  panic: fatal signal delivered at unknown() (unknown:0)  [git: 2.0.0-307-21-g05f1efe0] /var/motr/hax/m0trace.270375
    hare-hax[270375]: Motr panic: fatal signal delivered at unknown() unknown:0 (errno: 0) (last failed: none) [git: 2.0.0-307-21-g05f1efe0] pid: 270375  /var/motr/hax/m0trace.270375
    hare-hax[270375]: Motr panic reason: signo: 11
    ...
    Stack trace of thread 270397:
    #0  0x00007fce6d6ce7ff raise (libc.so.6)
    #1  0x00007fce6d6b8c35 abort (libc.so.6)
    #2  0x00007fce6554f453 m0_arch_panic (libmotr.so.2)
    #3  0x00007fce6553ec3d m0_panic (libmotr.so.2)
    #4  0x00007fce6554f49c sigsegv (libmotr.so.2)
    #5  0x00007fce6e1f6b20 __restore_rt (libpthread.so.0)
    #6  0x00007fce65bb540b adopt_motr_thread (libhax.cpython-36m-x86_64-linux-gnu.so)
    #7  0x00007fce6b87b09e ffi_call_unix64 (libffi.so.6)
    #8  0x00007fce6b87aa4f ffi_call (libffi.so.6)
    #9  0x00007fce6ba8d947 _ctypes_callproc.cold.53 (_ctypes.cpython-36m-x86_64-linux-gnu.so)
    #10 0x00007fce6ba91fd9 PyCFuncPtr_call (_ctypes.cpython-36m-x86_64-linux-gnu.so)
    #11 0x00007fce6e5a0d2b _PyObject_FastCallKeywords (libpython3.6m.so.1.0)
    #12 0x00007fce6e5a16f6 call_function (libpython3.6m.so.1.0)
    #13 0x00007fce6e5a21f8 _PyEval_EvalFrameDefault (libpython3.6m.so.1.0)
    #14 0x00007fce6e57e098 fast_function (libpython3.6m.so.1.0)
    #15 0x00007fce6e5a15b7 call_function (libpython3.6m.so.1.0)
    #16 0x00007fce6e5a21f8 _PyEval_EvalFrameDefault (libpython3.6m.so.1.0)
    #17 0x00007fce6e4ff112 _PyFunction_FastCallDict (libpython3.6m.so.1.0)
    #18 0x00007fce6e4ffeee _PyObject_FastCallDict (libpython3.6m.so.1.0)
    #19 0x00007fce6e511d80 method_call (libpython3.6m.so.1.0)
    #20 0x00007fce6e506bbb PyObject_Call (libpython3.6m.so.1.0)
    #21 0x00007fce6e5a406a _PyEval_EvalFrameDefault (libpython3.6m.so.1.0)
    #22 0x00007fce6e57e098 fast_function (libpython3.6m.so.1.0)
    #23 0x00007fce6e5a15b7 call_function (libpython3.6m.so.1.0)
    #24 0x00007fce6e5a21f8 _PyEval_EvalFrameDefault (libpython3.6m.so.1.0)
    #25 0x00007fce6e57e098 fast_function (libpython3.6m.so.1.0)
    #26 0x00007fce6e5a15b7 call_function (libpython3.6m.so.1.0)
    #27 0x00007fce6e5a21f8 _PyEval_EvalFrameDefault (libpython3.6m.so.1.0)
    #28 0x00007fce6e4ff112 _PyFunction_FastCallDict (libpython3.6m.so.1.0)
    #29 0x00007fce6e4ffeee _PyObject_FastCallDict (libpython3.6m.so.1.0)
    #30 0x00007fce6e511d80 method_call (libpython3.6m.so.1.0)
    #31 0x00007fce6e506bbb PyObject_Call (libpython3.6m.so.1.0)
    #32 0x00007fce6e612142 t_bootstrap (libpython3.6m.so.1.0)
    #33 0x00007fce6e5b8664 pythread_wrapper (libpython3.6m.so.1.0)
    #34 0x00007fce6e1ec14a start_thread (libpthread.so.0)
    #35 0x00007fce6d793f23 __clone (libc.so.6)

Solution: don't call adopt_motr_thread() - it's not needed
anymore in Motr since the Nikita's patch from 8 Sep 2020:
"implement __thread-based TLS" (commit 2900637d).

### CODE REVIEW

File: hax/hax/filestats.py [hunk 1]
```diff
@@ -47,7 +47,6 @@ def stop(self) -> None:
     def _execute(self, motr: Motr):
         try:
             LOG.info('filesystem stats updater thread has started')
-            motr.adopt_motr_thread()
```
Comment (suvratjoshi) [inline hax/hax/filestats.py:50]: `hare.spec` file changes should have no impact as the changes are in if/else for OS ver.
But for these changes which are OS independent, will need some testing on our current platform - CentOs 7.9.
Looks like testing was done for CentOs 8.
Just wanted to confirm if testing was done for 7.9 as well.

Thanks!

Comment (andriytk) [reply hax/hax/filestats.py:50]: I suppose the testing on 7.9 is done by our CI?

The `hax` was crashing on the very startup, so if it does not crash in startup in 7.9 during the CI - that's enough testing, I suppose.

Comment (suvratjoshi) [reply hax/hax/filestats.py:50]: ok.
Yes automated VM deployment on 7.9 here is passed.

**Review by suvratjoshi — APPROVED** [review]

**Comment by mssawant** [comment]
Somehow I couldn't find the commit,
```[520478@ssc-vm-1623:cortx-motr-1] git log 2900637d
fatal: ambiguous argument '2900637d': unknown revision or path not in the working tree.```

Approving, as vm deployment has passed.

**Review by mssawant — APPROVED** [review]

**Pull request merged**